python/nvtx/tests/test_basic.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 2020  NVIDIA Corporation.  All rights reserved.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import pickle
+
+import pytest
```

Comment:
I think this pytest dependency needs to be added in a `test_requires` block in `setup.py` and/or `pyproject.toml`

Reply:
We have a `test_requirements.txt` (https://github.com/NVIDIA/NVTX/pull/1/files#diff-6afeae61cfaf704a8f0db4151a014574) for this, but we can revisit later if people find it difficult to use that.